src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -267,7 +272,14 @@
 
 				// virtual list parameters
 				NUMBER_ITEMS_TO_ADD = 20,
-				LOAD_THRESHOLD = 10;
+				LOAD_THRESHOLD = 10,
+				overscrollRotaryHandler = null,
```

Comment:
I think we should move this out from constants.

Reply:
Done